crates/signer-ledger/src/signer.rs

```diff
@@ -278,20 +281,50 @@ impl LedgerSigner {
         self.sign_typed_data_with_separator(hash_struct, &domain.separator()).await
     }
 
+    // / Sign “auth data” per EIP-7702:
+    // / msg = keccak256(0x05 ‖ rlp([chain_id, address, nonce]))
+    #[cfg(feature = "eip7702")]
+    pub async fn sign_auth(
+    &self,
+    chain_id: ChainId,
+    delegate: Address,
+    nonce: u64,
```

Comment:
this is basically:

https://github.com/alloy-rs/eips/blob/ad74348a0c941401a8a42a7662f920274cbd54ab/crates/eip7702/src/auth_list.rs#L48-L56

which I think we can integrate here?